setup.py

```diff
@@ -47,7 +47,8 @@
     # incompatible with newer setuptools.
     RUN_REQUIRES.append('mpi4py >=3.1.1')
     SETUP_REQUIRES.append("mpi4py ==3.1.1; python_version<'3.11'")
-    SETUP_REQUIRES.append("mpi4py ==3.1.4; python_version>='3.11'")
+    SETUP_REQUIRES.append("mpi4py ==3.1.4; python_version=='3.11'")
```

Comment:
Should this be
```suggestion
    SETUP_REQUIRES.append("mpi4py ==3.1.4; python_version=='3.11.*'")
```

to catch all versions of py3.11?  I am always a bit fuzzy on the grammar of these version limits.

Reply:
Yes, you are right